README.md

```diff
@@ -22,6 +22,22 @@ data. For more information about the methodology and inquiries on
 running a pilot of the stress test in cooperation with 2DII, please
 contact <jacob@2degrees-investing.org>.
 
+## Prerequisites
```

Comment:
I would largely encourage removing this from the readme. The readme is developer facing, thus this information is self explanatory

Reply:
or rather remove points 1 und 2, point 3 makes sense also in a dev facing document.

Reply:
maybe we can just keep in point 3 and then extend the sentence below.. something like:
"We assume the user has successfully run at least the matching part of PACTA for Banks to produce the relevant project-specific raw input files. For a complete overview of the necessary prerequisites (including technical setup) please see [LINK]"

Reply:
sounds good to me